clio-server/src/test/scala/org/broadinstitute/clio/server/dataaccess/HttpElasticsearchDAOSpec.scala

```diff
@@ -41,11 +41,13 @@ class HttpElasticsearchDAOSpec
     val indexVersion1: ElasticsearchIndex[_] =
       new ElasticsearchIndex[ModelMockIndex](
         ModelMockIndex("test_index_update_type"),
+        Json.obj(),
```

Comment:
Shuffling the arg order so this goes after the `ElasticsearchFieldMapper` will let you avoid specifying it here (and everywhere else).